### PR TITLE
MYST-549 Terms & conditions

### PR DESCRIPTION
**Pull request by donce**
- terms flag for node
- summary terms text
- link to TERMS.md on github
- agreed-terms-and-conditions flag by default on docker_compose scripts

### CODE REVIEW

File: bin/server_docker/docker-entrypoint.sh [hunk 1]
```diff
@@ -11,5 +11,6 @@ exec /usr/bin/mysterium_server \
   --config-dir=$OS_DIR_CONFIG \
   --data-dir=$OS_DIR_DATA \
   --runtime-dir=$OS_DIR_RUN \
+  -agreed-terms-and-conditions=$AGREED_TERMS_AND_CONDITIONS \
```
Comment (Waldz) [inline bin/server_docker/docker-entrypoint.sh]: We just dropped environmental variables, lets not introduce them again

Comment (donce) [reply bin/server_docker/docker-entrypoint.sh]: Do you have some alternatives, which would work with Docker and Docker-Compose?

Comment (Waldz) [reply bin/server_docker/docker-entrypoint.sh]: We migrated to args style see `e2e/docker-compose.yml`

Comment (donce) [reply bin/server_docker/docker-entrypoint.sh]: If I understand correctly, declaring arg values in `e2e/docker-compose.yml` is fine, but we need **user input** for terms & conditions - this approach does not work for this.

Is this what you meant? Do you have other suggestions?

**Comment by ignasbernotas** [comment]
Aren't we supposed to store the terms and conditions once they accept like in Mysterion? Need to check up with #legal

File: bin/server_docker/docker-entrypoint.sh (hunk 1 above)
Comment (Waldz) [reply bin/server_docker/docker-entrypoint.sh]: It's user decision how to given input.
`docker run mysteriumnetwork/mysterium-node:latest --agreed-terms-and-conditions=...`

File: e2e/docker-compose.yml [hunk 2]
```diff
@@ -25,6 +25,8 @@ services:
       --discovery-address=http://discovery/v1
       --location.country=e2e-land
       --openvpn.port=3000
+    environment:
```
Comment (Waldz) [inline e2e/docker-compose.yml]: `command: --agreed-terms-and-conditions=04-06-2018`

File: README.md [hunk 3]
```diff
@@ -28,11 +28,12 @@ You can check latest docker node versions here: https://hub.docker.com/r/mysteri
 ### Installation
 
 Go to [docker](https://www.docker.com/) on how to get a recent docker version for Your OS.
-
+You will have to agree to the terms and conditions by appending flag '-agreed-terms-and-conditions'.
 ```bash
 sudo apt-get install docker-ce
-sudo docker run --cap-add NET_ADMIN --net host --publish "1194:1194" --name mysterium-node -d mysteriumnetwork/mysterium-node:latest
+sudo docker run --cap-add NET_ADMIN --net host --publish "1194:1194" --name mysterium-node -d mysteriumnetwork/mysterium-node:latest -agreed-terms-and-conditions
```
[the remark this answers is not in the document]
Comment (zolia) [reply README.md]: updated

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -37,13 +37,16 @@ docker-compose up broker
 
 # Start node
 bin/server_build
-bin/server_run
+bin/server_run -agreed-terms-and-conditions
```
Comment (Waldz) [inline CONTRIBUTING.md]: Suggest to remove this

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -37,13 +37,16 @@ docker-compose up broker
...
 
 # Client connects to node
 bin/client_build
 bin/client_run
 ```
 
+Note: to run server, you will have to accept terms & conditions.
```
Comment (Waldz) [inline CONTRIBUTING.md]: Suggest to remove this

File: docker-compose.yml [hunk 6]
```diff
@@ -17,6 +17,7 @@ services:
       - NET_ADMIN
     ports:
       - 11941:1194
+    command: --agreed-terms-and-conditions
```
[the remark this answers is not in the document]
Comment (zolia) [reply docker-compose.yml]: localnet does not starts nodes

**Pull request merged**